docs/Concepts/Merge.md

```diff
@@ -36,26 +39,27 @@ the consensus layer.
 Consensus clients serve [REST API](https://docs.teku.consensys.net/en/stable/Reference/Rest_API/Rest/) requests and
 communicate with each other in a peer-to-peer network.
 
-### Run a node
+## What happens during The Merge
+
+Before The Merge, the execution and consensus clients' configurations will be
+[updated](../HowTo/Upgrade/Prepare-for-The-Merge.md#update-besu) to listen for a certain total terminal difficulty (TTD)
+to be reached.
+
+The consensus layer will enable the Merge configuration (Bellatrix) before reaching the TTD.
+Once the execution layer blocks reach the TTD, the Beacon Chain will merge into Ethereum Mainnet, and Ethereum will move
+to a proof of stake network.
 
 After The Merge, a Mainnet node operator must run both an execution client and a beacon node at the same time.
 To become a validator, you must also run a validator client (either
 [in the same process as the beacon node](https://docs.teku.consensys.net/en/stable/HowTo/Get-Started/Run-Teku/#start-the-clients-in-a-single-process)
 or [separately](https://docs.teku.consensys.net/en/stable/HowTo/Get-Started/Run-Teku/#run-the-clients-separately).
 
-You can
-[test Besu as an execution client on the Merge testnet with Teku](../Tutorials/Merge-Testnet.md).
-
-You can
-[run Besu as an execution client on Ethereum Mainnet with Teku](https://docs.teku.consensys.net/en/stable/HowTo/Get-Started/Connect/Connect-To-Mainnet/).
-
-Consensus clients compatible with Besu include:
+After The Merge, in addition to validators earning rewards for performing
+[validator duties](https://docs.teku.consensys.net/en/stable/Concepts/Proof-of-Stake/),
+[fee recipients](https://docs.teku.consensys.net/en/latest/HowTo/Prepare-for-The-Merge/#configure-the-fee-recipient)
+will also earn rewards for the inclusion of execution layer transactions.
 
-- [Teku] (recommended)
-- [Lighthouse](https://lighthouse.sigmaprime.io/)
-- [Lodestar](https://lodestar.chainsafe.io/)
-- [Prysm](https://github.com/prysmaticlabs/prysm)
-- [Nimbus](https://nimbus.team/)
+You can [prepare Besu for The Merge](../HowTo/Upgrade/Prepare-for-The-Merge.md).
```

Comment:
Perhaps as in the earlier comment mention that you must upgrade and configure Besu to prepare for the merge.